lib/improver/spotdata/neighbour_finding.py

```diff
@@ -192,6 +192,10 @@ def fast_nearest_neighbour(cube, sites, orography=None):
             dz_site_grid = 0.
             if orography is not None:
                 dz_site_grid = altitude - orography[i_latitude, j_longitude]
+            # If site altitude set with fill value, assume it is on surface of
+            # of model orography at equivalent altitude to nearest neighbour.
```

Comment:
Comment changed.
If statement fixed.
Using an init defined fill value, but currently defined in two places to prevent an inter dependence between the modules. Constants doesn't seem the right place as it's not a physical constant, but I agree somewhere central would be good. common_functions perhaps?